src/main/java/com/neprozorro/model/Participant.java

```diff
@@ -0,0 +1,34 @@
+package com.neprozorro.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "participant")
+public class Participant {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @Column(name = "id")
+    private UUID id;
+
+    /** ЄДРПУО **/
+    @Column(name = "sgrpul")
+    private String sgrpul;
```

Comment:
Maybe we should rename sgrpul to edrpou.

Reply:
Done. I agree!

Reply:
is the comment enough or you should've changed it to erdpou?